bff-node/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "bff-node",
```

Comment:
should we give the app a better name, "bff-altinn-inbox"? Not sure if it is a better name, but bff-node might be to general. 😊